app/controllers/portal_controller.rb

```diff
@@ -170,7 +174,8 @@ def exhibition_url_prefix(locale)
   def exhibition_url
     creator_name = Rails.application.config.x.exhibitions.annotation_creator_name
     uri_query = "(#{exhibition_url_prefix(locale)}* OR #{exhibition_url_prefix(I18n.default_locale)}*)"
-    annotations = document_annotations(doc_id, creator_name: creator_name, link_resource_uri: uri_query, limit: 100)
+    annotations = document_annotations(doc_id, limit: 100,
+                                               qf: { creator_name: creator_name, link_resource_uri: uri_query })
```

Comment:
these are now missing the escaping, and the target_record_id is being skipped altogether